[Cache] Adds clarification for cache rules fields and purge (#19096)
cloudflare/cloudflare-docs: src/content/docs/cache/how-to/cache-rules/settings.mdx

## Fields

The fields available for Cache Rule matching expressions in the **Expression Builder** are:

- URI Full - `http.request.full_uri`
- URI - `http.request.uri`
- URI Path - `http.request.uri.path`
- URI Query String - `http.request.uri.query`
- Cookie - `http.cookie`
- Hostname - `http.host`
- Referer - `http.referer`
- SSL/HTTPS - `ssl`
- User Agent - `http.user_agent`
- X-Forwarded-For - `http.x_forwarded_for`
- Request Headers - `http.request.headers`
- Cookie value of - `http.request.cookies`
- File extension - `http.request.uri.path.extension`

> If you select the [**Edit expression**](/ruleset-engine/rules-language/expressions/edit-expressions/#expression-editor) option, this is the complete list of available [Fields](/ruleset-engine/rules-language/fields/).

Reviewer: ```suggestion
If you select the [**Edit expression**](/ruleset-engine/rules-language/expressions/edit-expressions/#expression-editor) option, you can enter any of the [available fields](/ruleset-engine/rules-language/fields/).
```

:::note

> The fields mentioned above support all types of purge. However, these [fields](/ruleset-engine/rules-language/fields/) support purge by tags, prefix (URL), hostname, and everything but are not compatible with [Single File Purge](/cache/how-to/purge-cache/purge-by-single-file/). 

Reviewer: ```suggestion
The fields mentioned above support all types of purge. However, the [remaining available fields](/ruleset-engine/rules-language/fields/) support purge by tags, prefix (URL), hostname, and everything but are not compatible with [single file purge](/cache/how-to/purge-cache/purge-by-single-file/). 
```
Nits:
I'd replace "these fields".
Also, I don't think we're capitalizing "single file purge" in the target page.
